### PR TITLE
Fix Cover.sh

### PR DESCRIPTION
**Pull request by kron4eg**
**What this PR does / why we need it**:
Fix this script to be more specific to test only `./pkg/...` to avoid letting go test see the ./hack/tools.go

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Documentation**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: hack/coverage.sh [hunk 1]
```diff
@@ -44,18 +44,20 @@ generate_cover_data() {
...
 
 show_cover_report() {
-  echo "Generating ${1} report"
-  go tool cover -${1}="$profile" ${2}
+  echo "Generating $1 report"
+  go tool cover -"$1"="$profile" "$2"
 }
 
-generate_cover_data $(go list ./...)
+generate_cover_data "$(go list ./pkg/...)"
```
Comment (xrstf) [inline hack/coverage.sh]: We do have tests in `./cmd/...` as well.

**Review by xrstf — COMMENTED** [review]
Should we include `cmd` as well?

File: hack/coverage.sh (hunk 1 above)
Comment (kron4eg) [reply hack/coverage.sh]: 👍

**Review by xrstf — APPROVED** [review]
/approve

**Comment by kron4eg** [comment]
rebased with git push force

**Review by xrstf — APPROVED** [review]
/approve

**Pull request merged**